File: 1010.c
//Exercicio 1010

#include <stdio.h>

int main() {


    int cod1, pecas1, cod2, pecas2;
    double valor1, valor2, total;

    scanf ("%d %d %lf", &cod1, &pecas1, &valor1);
    scanf ("%d %d %lf", &cod2, &pecas2, &valor2);

    total = pecas1*valor1 + pecas2*valor2;

    printf ("VALOR A PAGAR: R$ %.2lf\n", total);

    return 0;
}

File: Python/1010.py
cod1 = int(input())
pecas1 = int(input())
valor1 = float(input())

cod2 = int(input())
pecas2 = int(input())
valor2 = float(input())

total = pecas1*valor1 + pecas2*valor2

print('VALOR A PAGAR: R$ {}\n' .format(round(total, 2)))
